### PR TITLE
Updated Emacs cheatsheet

### PR DESCRIPTION
**Pull request by CsBigDataHub**
Updated Emacs cheatsheet

### CODE REVIEW

File: emacs.md [hunk 1]
```diff
@@ -1,27 +1,363 @@
...
+| C-r  | Start a reverse search. |
+| M-%  | Query-replace           |
+{: .-shortcuts}
+
+## Undo / Redo
+
+| Keys  | Description |
+| ----- | ----------- |
+| C-x u | Undo        |
+| C-\_  | Undo        |
```
Comment (dekdaas) [inline emacs.md:119]: Extra underscore here

File: emacs.md [hunk 2]
```diff
@@ -1,27 +1,363 @@
...
+{: .-shortcuts}
+
+## ParEdit-Mode
+
+| Keys             | Description                                                             |
+| ---------------- | ----------------------------------------------------------------------- |
+| C-right          | Slurp; move closing parenthesis to the right to include nextexpression. |
+| C-left           | Barf; move closing parenthesis to the left to exclude lastexpression.   |
+| C-M-f            | Move to the opening/closing parenthesis.                                |
+| C-M-b            | Move to the opening/closing parenthesis.                                |
```
Comment (dekdaas) [inline emacs.md:151]: Maybe differentiate as 'next' and 'previous'?

File: emacs.md (hunk 1 above)
Comment (dekdaas) [inline emacs.md:119]: Backslash is extraneous/potentially confusing

**Comment by CsBigDataHub** [comment]
@dekdaas friendly ping.

**Comment by CsBigDataHub** [comment]
@dekdaas @rstacruz Is there anything I need to do to get this PR approved?

**Pull request closed without merging**